README.md

```diff
@@ -1,20 +1,104 @@
 # Moped
-### A comprehensive project tracking platform for Austin's mobility projects 
-🛵 🗺 👷 🚌 🚧 🚴 📊  
 
-### Resources
+### A comprehensive project tracking platform for Austin's mobility projects
+
+🛵 🗺 👷 🚌 🚧 🚴 📊
+
+## Resources
+
 - [About](https://austinmobility.io/products/5086)
 - [Documentation](https://atd-dts.gitbook.io/moped-documentation)
 - [Backlog](https://github.com/cityofaustin/atd-data-tech/issues?q=is%3Aopen+label%3A%22Product%3A+Moped%22+-label%3A%22Workgroup%3A+TPW%22)
 
-### Environments
+## Environments
+
 - Production: [https://mobility.austin.gov/moped/](https://mobility.austin.gov/moped/)
 - Staging: [https://moped.austinmobility.io/moped/](https://moped.austinmobility.io/moped/)
 - Test: [https://moped-test.austinmobility.io/moped/](https://moped-test.austinmobility.io/moped/)
 
-For resources and updates, see the [Mobility Project Database project index](https://github.com/cityofaustin/atd-data-tech/issues/307). 
+For resources and updates, see the [Mobility Project Database project index](https://github.com/cityofaustin/atd-data-tech/issues/307).
 
 ![Post-it notes of users' desired Moped functionality and outcomes](https://user-images.githubusercontent.com/1463708/62583080-58614e80-b874-11e9-850d-2a8bda07c0fc.jpeg)
 
+## Getting Started for Developers
```

Comment:
Could you update this heading to say **Quick start** like the [VZ readme](https://github.com/cityofaustin/vision-zero/blob/main/README.md)? I think it would be nice to align the way we document the two projects as much as possible.